Tools/scripts/sitl-on-hardware/sitl-on-hw.py

```diff
@@ -66,6 +66,8 @@ def sohw_path(fname):
     defaults_write(open(args.defaults,"r").read() + "\n")
 
 if args.simclass:
+    if args.simclass == 'Glider':
+        hwdef_write("define AP_SIM_GLIDER_ENABLED 1\n")
```

Comment:
Interesting approach.  We'll see if it works.

Alternate approach would be to enable the feature based on `AP_SIM_ENABLED` and always have it in there, even if it isn't actually usable.

Reply:
Maybe I should have mentioned, it does work on my local tests.
Unless you had some other testing in mind.